### PR TITLE
[PLATFORM-310] Docs Code Snippeting

### PR DESCRIPTION
**Pull request by fonty1**
See /docs/tutorials for it in action. This PR includes the first 3 Tutorial content pieces. 

Code highlighting snippets via [react-syntax-highlighter](https://github.com/conorhastings/react-syntax-highlighter).

Larger code snippets in their own files that are imported into the various `content.mdx`

PR contains some generic Docs styling changes as I didn't merge the original in time but nothing too crazy.

### CODE REVIEW

**Comment by juhah** [comment]
I've added a shared `CodeSnippet` component in #223 based on your component, that could be taken into use here as well.

File: app/src/docs/components/DocsLayout/Navigation/index.jsx [hunk 1]
```diff
@@ -20,17 +20,17 @@ type Props = {
 }
 
 type State = {
-    collapsed: boolean,
+    compressed: boolean,
```
[the remark this answers is not in the document]
Comment (fonty1) [reply app/src/docs/components/DocsLayout/Navigation/index.jsx]: I actually forgot the change collapsed -> compressed in the first place. Sorry about that, getting confusing over multiple PRs!

File: app/src/docs/components/TutorialsPage/codeSnippets.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    req = requests.post(url, params=headers, data=body)
+    token = req.json()['access_token']
+    LOG.info("Get user token successfully: %s", token)
+    return token
+except:
+    import traceback
+    LOG.error("Unable to get user token")
+    traceback.print_exc()
+    return None
+def get_user_steps(access_token):<br>    pass
```
[the remark this answers is not in the document]
Comment (fonty1) [reply app/src/docs/components/TutorialsPage/codeSnippets.js]: Which part do you mean sorry? This is copy paste from a user submitted tutorial content piece.

[the remark this answers is not in the document]
Comment (tumppi) [reply app/src/docs/components/TutorialsPage/codeSnippets.js]: Ah, sorry. GitHub is pretty confusing sometimes picking the "context" for comments. I mean the `<br>` :)

**Comment by fonty1** [comment]
> I've added a shared `CodeSnippet` component in #223 based on your component, that could be taken into use here as well.

@juhah it's now using the shared CodeSnippet™️ component, please take another look. Also included some minor styling fixes.

**Pull request merged**